Test/Testable.lean

```diff
@@ -24,6 +24,5 @@ instance : SampleableExt MyType :=
     let xyDiff ← SampleableExt.interpSample Nat
     return ⟨x, x + xyDiff, by omega⟩
 
--- TODO: this is a noisy test.
--- We can't use `#guard_msgs` because the number of attempts to non-deterministic.
 #eval Testable.check <| ∀ a b : MyType, a.y ≤ b.x → a.x ≤ b.y
```

Comment:
It's a bit unfortunate there are all these tests showing that deriving is possible but there is no test that shows them actually find a counter example so people that touch your code in the future will always have to inspect the generated code manually. Could you add a few tests like that?